MapboxNavigation/UserCourseView.swift

```diff
@@ -146,6 +176,51 @@ public class UserPuckCourseView: UIView, CourseUpdatable {
     }
 }
 
+class UserApproximateStyleKitView: UIView {
```

Comment:
Please also make sure to run some tests with different use cases in active guidance: e.g. in Overview/Follow modes, different zoom levels, Day/Night styles, vanishing route line enabled/disabled.

Reply:
Also when it comes to testing I also recommend moving to `Xcode 12.0`. Thanks to @1ec5 you can now easily build Carthage dependencies by running `$ ./scripts/wcarthage.sh bootstrap --use-netrc --use-ssh --platform ios`.

Reply:
Thanks for the suggestion! I'll test the iOS14 with that.